packages/openactive-broker-microservice/src/state.js

```diff
@@ -126,24 +187,21 @@ const state = {
 };
 
 /**
+ * All Opportunity IDs that are considered "locked" (because they have already
+ * been used in a test) for the specified [Test Dataset](https://openactive.io/test-interface/#datasets-endpoints).
+ *
  * @param {string} testDatasetIdentifier
+ * @returns {Set<string>}
  */
-function getTestDataset(testDatasetIdentifier) {
-  if (!state.testDatasets.has(testDatasetIdentifier)) {
-    state.testDatasets.set(testDatasetIdentifier, new Set());
+function getLockedOpportunityIdsInTestDataset(testDatasetIdentifier) {
```

Comment:
Try to make it clear that what's being received isn't a test dataset in general, but info about what opportunities are locked in that test dataset